src/mesh/Mesh.hpp

```diff
@@ -258,6 +267,18 @@ class Mesh : public PropertyContainer, private boost::noncopyable
     _globalNumberOfVertices = num;
   }
 
+  /// Returns a vector of connected ranks
+  std::vector<int> & getInitialCommunicationMap()
```

Comment:
Probably not needed

Reply:
we need this, because the initialConnectionMap is private! But the name should be changed accordingly!